README.md

```diff
@@ -20,6 +40,32 @@ Note, environment files will be created on *postinstall* (see [scripts/setup-env
 $ yarn dev
 ```
 
+### Build
+
+```sh
+# Build all the packages
+$ yarn build
+
+# Build the "code-du-travail-frontend"
+$ yarn workspace code-du-travail-frontend build
+
+# Build the "<package-name>"
+$ yarn workspace <package-name> build
```

Comment:
Why not put  these one (genéric) before the example

Reply:
done